.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,67 +1,72 @@
-[Short description explaining the high-level reason for the pull request]
+_[Enter an explanation of what the pull request does and why.]_
 
-## Additions
 
--
-
-## Removals
-
--
+---
 
-## Changes
```

Comment:
I'm suggesting the removal of these three sections, because they often feel like a chore of restating what I already wrote in the description above.

Reply:
I actually use this section quite a bit (compared to the others that I treated as guidance and delete). I add things to the `Changes` I'd see as bullet points suitable for a commit message (particularly useful if commits have been squashed) and I elaborate on anything necessary in the description at the top—if applicable—but often only use the bullet point section. What if we just had a `Changes` section at the top with a beginning bullet point? Or if we had that and a space for a description above it. I feel like chucking all of these is discouraging summarization of changes. 

(FWIW these sections grew out of https://keepachangelog.com/en/1.0.0/#how I'm pretty sure)

Reply:
I restored them in d3131aca2f5fde0f48ee0a4d3c8f410a120f8acb.